.github/workflows/github-ci-beaverhours.yml

```diff
@@ -0,0 +1,56 @@
+# CI Pipeline adapted for BeaverHours integration from github-ci-template.yml found at https://github.com/OfficeDev/TeamsFx/blob/main/docs/cicd_insider/README.md
+
+name: 'Continuous Integration'
+on:
+  # When pull requests targeting the main branch created.
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  buildAndTest:
+    runs-on: ubuntu-latest
+    steps:
+      # Setup environment.
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Checkout the code
+        uses: actions/checkout@v2
+ 
+      - name: Check check code formatting
+        uses: creyD/prettier_action@v4.1.1
+        with:
+          # This part is also where you can pass other options, for example:
+          # prettier_options: --write **/*.{js,md,ts}   # actually formats 
+          prettier_options: --check .
+
+      # -----Build and test bot----- 
+      # Build the project.
+      # The way to build the current project depends on how you scaffold it.
+      # Different folder structures require different commands set.
+      # 'npm ci' is used here to install dependencies and it depends on package-lock.json.
+      # If you prefer to use 'npm ci', please make sure to commit package-lock.json first, or just change it to 'npm install'.
+      - name: Build the project
+        run: cd bot && npm install && npm run build && cd -
+
+      # Run unit test.
+      # Currently, no opinionated solution for unit test provided during scaffolding, so,
+      # set up any unit test framework you prefer (for example, mocha or jest) and update the commands accordingly in below.
+      # - name: Run Unit Test
+        # run: cd bot && npm run test && cd -
+ 
+      # # Tab integration tests are off as tab development is a stretch goal.
+      # # ----- Build and test tab-----
+      # - name: Build the project
+      #   # run: cd tabs && npm ci && npm run build && cd -
+      #   run: cd tabs && npm install & npm run build && cd -
```

Comment:
You can delete all of this commented code, as we'll be able to add it back when we need to implement it.

Reply:
Removed the tab testing code and formatted the project files using Prettier as suggested. Good call on the vscode extension. Let me know if these changes are sufficient!